spec/01-unit/31-simdjson/01-cjson_compatibility_spec.lua

```diff
@@ -0,0 +1,284 @@
+local helpers = require("spec.helpers")
+local cjson = require("cjson.safe")
+local simdjson = require("resty.simdjson")
+
+
+local deep_sort = helpers.deep_sort
+
+
+describe("[cjson compatibility] examples from cjson repo", function ()
+
+  local strs = {
+[[
+{
+    "glossary": {
+        "title": "example glossary",
+                "GlossDiv": {
+            "title": "S",
+                        "GlossList": {
+                "GlossEntry": {
+                    "ID": "SGML",
+                                        "SortAs": "SGML",
```

Comment:
the format alignment

Reply:
No, we will keep the original data format.